### PR TITLE
dxGantt: header filter

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/_hidden/dxGanttHeaderFilterTexts/cancel.md [hunk 1]
```diff
@@ -5,7 +5,6 @@ default: 'Cancel'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies text for the "Cancel" button in the popup window.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/_hidden/dxGanttHeaderFilterTexts/cancel.md:8]: By "text" do you mean the button's caption?

File: api-reference/_hidden/dxGanttHeaderFilter/searchTimeout.md [hunk 2]
```diff
@@ -5,7 +5,6 @@ default: 500
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a delay in milliseconds between typing a search string and the search execution.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/_hidden/dxGanttHeaderFilter/searchTimeout.md:8]: Do you mean a delay between the last change in the search box and the effect of that change?

File: api-reference/_hidden/dxGanttHeaderFilterTexts/ok.md [hunk 3]
```diff
@@ -5,7 +5,6 @@ default: 'Ok'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies text for the "OK" button in the popup window.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/_hidden/dxGanttHeaderFilterTexts/ok.md:8]: Does this property allows you to change the button's initial caption? That is, "OK"...

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
The rest looks good to me.

File: api-reference/_hidden/dxGanttHeaderFilter/searchTimeout.md (hunk 2 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxGanttHeaderFilter/searchTimeout.md:8]: Yes.

File: api-reference/_hidden/dxGanttHeaderFilterTexts/cancel.md (hunk 1 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxGanttHeaderFilterTexts/cancel.md:8]: Yes.

File: api-reference/_hidden/dxGanttHeaderFilterTexts/ok.md (hunk 3 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxGanttHeaderFilterTexts/ok.md:8]: Yes

**Review by LanaDX — APPROVED** [review]

**Pull request merged**